README.md

```diff
@@ -12,8 +11,137 @@ This is a checker wrapper lib.
 - testing output
 
 
-#### How to: 
+### Quick Start
+#### Examples
+* [Minimal working example](https://github.com/HackerDom/Gornilo/blob/master/gornilo/examples/checker.py) 
```

Comment:
Use relative pathes instead of absolute:
`[Minimal working example](examples/checker.py)`

Same for abstract_checker.py